cmd/config-seed/Attribution.txt

```diff
@@ -63,6 +63,9 @@ https://github.com/cenkalti/backoff/blob/master/LICENSE
 hashicorp/consul 1.1.0 (Mozilla Public License 2.0) - https://github.com/hashicorp/consul
 https://github.com/hashicorp/consul/blob/master/LICENSE
 
+Nested consul packages are not separately licensed:
+github.com/hashicorp/consul/api
```

Comment:
please specify what license this package is under, not just the github link. See other attributions in the file for an example

Reply:
All of that information is already there.  The attribution checker just complains that the nested package consul/api isn't covered.  Please look at the lines above all of my attribution.txt changes, thanks

Reply:
ah could you file a bug for me to fix the attribution checker for that then? 

Also, does the checker pass if you delete the above line and just have the LICENSE for the nested api package? I seem to remember us switching to just using the api package so we should be able to delete the root level package from the checker.

Reply:
@anonymouse64 I think I've made the changes you suggested--can you check to make sure I understood you properly?

Reply:
I will file an issue right away